lib/java/test/org/apache/thrift/transport/TestTFastFramedTransport.java

```diff
@@ -19,6 +19,8 @@
 package org.apache.thrift.transport;
 
 public class TestTFastFramedTransport extends TestTFramedTransport {
+  protected final static int INITIAL_CAPACITY = 50;
+
   @Override
   protected TTransport getTransport(TTransport underlying) {
     return new TFastFramedTransport(underlying, 50, 10 * 1024 * 1024);
```

Comment:
I think 50 was supposed to be changed to ``INITIAL_CAPACITY`` here, something I could do as part of the merge.